scripts/release.helpers.js

```diff
@@ -0,0 +1,126 @@
+const readline = require('node:readline/promises');
+
+const { exec } = require('child-process-promise');
+
+async function checkGitStatus() {
+  const result = await runExec('git status --porcelain', {
+    silent: true,
+  });
+  const changes = result.stdout.toString().trim();
+  if (changes) {
+    console.error('🟠 There are unstaged git files. Please commit or revert them and try again.');
+    process.exit(1);
+  }
+}
+
+async function checkNpmAuth() {
+  console.log('Checking NPM Authentication...');
+  try {
+    const result = await runExec('npm whoami');
+    const username = result.stdout.toString().trim();
+    if (username !== 'remoteoss') {
+      console.log('🟠 You need to be logged to NPM as "remoteoss". Run "npm adduser"');
+      process.exit(1);
+    }
+  } catch (e) {
+    process.exit(1);
+  }
+}
+
+async function askForText(question) {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+  const answer = await rl.question(question);
+  rl.close();
+  return answer;
+}
+
+async function askForConfirmation(question) {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+  const answer = await rl.question(`${question} (Y/n)`);
+  const normalizedAnswer = answer.trim().toLowerCase();
+
+  rl.close();
+
+  if (normalizedAnswer === 'y' || normalizedAnswer === 'yes') {
+    console.log('Confirmed! Proceeding...');
+    return 'yes';
+  }
+  if (normalizedAnswer === 'n' || normalizedAnswer === 'no') {
+    console.log('Cancelled. Exiting...');
+    return 'no';
+  }
+  console.log('Invalid input. Please enter Y or n.');
+  return askForConfirmation();
```

Comment:
Need to pass in `question` here again.

Reply:
Excellent catch, fixed https://github.com/remoteoss/json-schema-form/pull/3/commits/6019938c955c16636ea02e15a01640572db02563